docs/source/developer_guide/guides/10_modular_pipeline_digital_fingerprinting.md

```diff
@@ -514,3 +514,134 @@ def write_to_file(builder: mrc.Builder):
 ```
 
 For a complete reference, refer to: [Write to File](../../modules/core/write_to_file.md)
+
+## Running Example Modular DFP Pipelines
+
+The following are steps to run modular DFP pipelines with example Azure and Duo datasets.
+
+### System requirements
+* [Docker](https://docs.docker.com/get-docker/) and [docker-compose](https://docs.docker.com/compose/) installed on the host machine​
+* Supported GPU with [nvidia-docker runtime​](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
+
+> **Note:**  For GPU Requirements refer to [getting_started](../../getting_started.md#requirements)
+
+### Building the services
+From the root of the Morpheus repo, run:
+```bash
+cd examples/digital_fingerprinting/production
+docker compose build
+```
+
+> **Note:** This requires version 1.28.0 or higher of Docker Compose, and preferably v2. If you encounter an error similar to:
+>
+> ```
+> ERROR: The Compose file './docker-compose.yml' is invalid because:
+> services.jupyter.deploy.resources.reservations value Additional properties are not allowed ('devices' was
+> unexpected)
+> ```
+>
+> This is most likely due to using an older version of the `docker-compose` command, instead re-run the build with `docker compose`. Refer to [Migrate to Compose V2](https://docs.docker.com/compose/migrate/) for more information.
+
+### Downloading the example datasets
+First, we will need to install `s3fs` and then run the `examples/digital_fingerprinting/fetch_example_data.py` script.  This will download the example data into the `examples/data/dfp` dir.
+
+From the Morpheus repo, run:
+```bash
+pip install s3fs
+python examples/digital_fingerprinting/fetch_example_data.py all
+```
+
+### Morpheus Pipeline
+From the `examples/digital_fingerprinting/production` dir, run:
+```bash
+docker compose run morpheus_pipeline bash
+```
+To run the DFP pipelines with the example datasets within the container, run:
+
+* Duo Training Pipeline
+   ```bash
+   python dfp_modules_batch_pipeline.py \
+    --log_level DEBUG \
+    --use_cpp=true \
+    --source duo \
+    --start_time "2022-08-01" \
+    --duration "60d" \
+    --train_users generic \
+    --input_file "./control_messages/duo_payload_training.json"
+   ```
+
+* Duo Inference Pipeline
+   ```bash
+   python dfp_modules_batch_pipeline.py \
+    --log_level DEBUG \
+    --use_cpp=true \
+    --source duo \
+    --start_time "2022-08-30" \
+    --input_file "./control_messages/duo_payload_inference.json"
+   ```
```

Comment:
Can we increase the indent here?
```suggestion
   ```bash
   python dfp_modules_batch_pipeline.py \
       --log_level DEBUG \
    --use_cpp=true \
    --source duo \
    --start_time "2022-08-30" \
    --input_file "./control_messages/duo_payload_inference.json"
   ```
```